
<!DOCTYPE html>

<html lang="en">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="generator" content="Docutils 0.19: https://docutils.sourceforge.io/" />

    <title>VA Data Commons &#8212; va-doc 1.0.0 documentation</title>
    <link rel="stylesheet" type="text/css" href="_static/pygments.css" />
    <link rel="stylesheet" type="text/css" href="_static/alabaster.css" />
    <script data-url_root="./" id="documentation_options" src="_static/documentation_options.js"></script>
    <script src="_static/doctools.js"></script>
    <script src="_static/sphinx_highlight.js"></script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
   
  <link rel="stylesheet" href="_static/custom.css" type="text/css" />
  
  
  <meta name="viewport" content="width=device-width, initial-scale=0.9, maximum-scale=0.9" />

  </head><body>
  

    <div class="document">
      <div class="documentwrapper">
        <div class="bodywrapper">
          

          <div class="body" role="main">
            
  <section id="va-data-commons">
<h1><strong>VA Data Commons</strong><a class="headerlink" href="#va-data-commons" title="Permalink to this heading">¶</a></h1>
<p>The VA Data Commons supports the research and analysis of US military
Veteran medical and genomic data and aims to accelerate scientific
discovery and development of therapies, diagnostic tests, and other
technologies for improving the lives of Veterans and beyond. The data
commons features GWAS analyses on harmonized data.</p>
</section>
<section id="table-of-contents">
<h1>Table of Contents<a class="headerlink" href="#table-of-contents" title="Permalink to this heading">¶</a></h1>
<ul class="simple">
<li><p><a class="reference external" href="#va-data-commons">VA Data Commons</a></p></li>
<li><p><a class="reference external" href="#table-of-contents">Table of Contents</a></p></li>
<li><p><a class="reference external" href="#getting-started">Getting Started</a></p>
<ul>
<li><p><a class="reference external" href="#login-page">Login Page</a></p>
<ul>
<li><p><a class="reference external" href="#va-data-commons-documentation">VA Data Commons
Documentation</a></p></li>
<li><p><a class="reference external" href="#email-support">Email Support</a></p></li>
</ul>
</li>
</ul>
</li>
<li><p><a class="reference external" href="#data-access-and-analysis">Data Access and Analysis</a></p>
<ul>
<li><p><a class="reference external" href="#workspace">Workspace</a></p></li>
<li><p><a class="reference external" href="#apps">Apps</a></p></li>
<li><p><a class="reference external" href="#ohdsi-atlas">OHDSI Atlas</a></p>
<ul>
<li><p><a class="reference external" href="#steps-to-generate-a-cohort">Steps to Generate a Cohort</a></p></li>
</ul>
</li>
<li><p><a class="reference external" href="#gen3-gwas">Gen3 GWAS</a></p></li>
<li><p><a class="reference external" href="#genome-wide-association-studies-gwas-steps">Genome-Wide Association Studies (GWAS)
Steps</a></p></li>
<li><p><a class="reference external" href="#gwas-results">GWAS Results</a></p></li>
</ul>
</li>
<li><p><a class="reference external" href="#frequently-asked-questions">Frequently Asked Questions</a></p>
<ul>
<li><p><a class="reference external" href="#where-can-i-find-more-documentation-on-the-ohdsi-atlas-app">Where can I find more documentation on the OHDSI Atlas
app?</a></p></li>
<li><p><a class="reference external" href="#what-are-harmonized-variables">What are harmonized
variables?</a></p></li>
<li><p><a class="reference external" href="#what-will-be-included-in-the-downloadable-package-at-the-end-of-a-gwas">What will be included in the downloadable package at the end of a
GWAS?</a></p></li>
<li><p><a class="reference external" href="#how-long-will-my-gwas-take">How long will my GWAS take?</a></p></li>
<li><p><a class="reference external" href="#why-does-the-workspaces-page-give-me-an-error">Why does the workspaces page give me an
error?</a></p></li>
</ul>
</li>
</ul>
</section>
<section id="getting-started">
<h1>Getting Started<a class="headerlink" href="#getting-started" title="Permalink to this heading">¶</a></h1>
<section id="login-page">
<h2><strong>Login Page</strong><a class="headerlink" href="#login-page" title="Permalink to this heading">¶</a></h2>
<p>Users will log in with their VA PIV credentials. Start by visiting the
<a class="reference external" href="https://va.data-commons.org/login">login page</a> and sign in with your
organizational account. After successfully logging in, your username
will appear in the upper right-hand corner of the page. Click this link
for more information about using the <a class="reference external" href="https://www.oit.va.gov/resources/remote-access/cag/windows/index.cfm?#piv">PIV
login</a>.</p>
<p>Once logged in, the user will see there are several options in the upper
right hand corner of the <a class="reference external" href="https://va.data-commons.org/">home screen</a>.
In the blue bar at the top of the page, there is a link to VA Data
Commons Documentation (the page you are currently reading), Email
Support, your username and a log out button. Below these options, there
are tabs for the Workspace, the Apps, and the Profile pages. Clicking on
the VA Data Commons logo in the upper left hand corner of the screen
will always return you to the home page of the data commons.</p>
<section id="va-data-commons-documentation">
<h3><strong>VA Data Commons Documentation</strong><a class="headerlink" href="#va-data-commons-documentation" title="Permalink to this heading">¶</a></h3>
<p>The button for VA Data Commons Documentation takes you to this page.</p>
</section>
<section id="email-support">
<h3><strong>Email Support</strong><a class="headerlink" href="#email-support" title="Permalink to this heading">¶</a></h3>
<p>If you need help, the Email Support button allows you to send a message
to our help desk at <a class="reference external" href="mailto:MVPLOI&#37;&#52;&#48;va&#46;gov">MVPLOI<span>&#64;</span>va<span>&#46;</span>gov</a>. You may expect a response within 2
business days.</p>
</section>
</section>
</section>
<section id="data-access-and-analysis">
<h1>Data Access and Analysis<a class="headerlink" href="#data-access-and-analysis" title="Permalink to this heading">¶</a></h1>
<section id="workspace">
<h2><strong>Workspace</strong><a class="headerlink" href="#workspace" title="Permalink to this heading">¶</a></h2>
<p>Currently, workspaces are not available, and therefore you may see an
error when attempting to log into this page. Generally, workspaces are
secure data analysis environments in the cloud that can access data from
one or more data resources. Workspaces may include Jupyter notebooks and
JupyterLab, Python and RStudio. For more information about the Gen3
Workspace, you may refer to <a class="reference external" href="https://gen3.org/products/workspaces/">Gen3
Workspaces</a> and <a class="reference external" href="https://gen3.org/resources/user/analyze-data/">Data Analysis
in a Gen3 Data
Commons</a>.</p>
</section>
<section id="apps">
<h2><strong>Apps</strong><a class="headerlink" href="#apps" title="Permalink to this heading">¶</a></h2>
<p>The <a class="reference external" href="https://va.data-commons.org/analysis">Apps page</a> displays the
option of three Apps: OHDSI Atlas, Gen3 GWAS and GWAS Results. Using
these Apps, a user may perform a genomic analysis on available data from
projects that they have access to.</p>
</section>
<section id="ohdsi-atlas">
<h2><strong>OHDSI Atlas</strong><a class="headerlink" href="#ohdsi-atlas" title="Permalink to this heading">¶</a></h2>
<p>ATLAS is an open source software application developed as a part of
<a class="reference external" href="https://www.ohdsi.org/">OHDSI</a> community intended to provide a
unified interface to patient level data and analytics. Atlas software
used to define cohorts, typically dichotomous variables, for analysis.</p>
<p>ATLAS currently includes functionality for searching and navigating the
vocabulary within the OMOP Common Data Model (CDM). In addition to the
search and navigation capabilities, it also provides features to curate
and export custom sets of concept identifiers for use in cohort
definitions. These will automatically populate on the Gen3 GWAS App. In
general, ATLAS is an analytics platform that can be used to perform
analyses across one or more observational databases which have been
standardized to the OMOP Common Data Model V5 and can facilitate
exchange of analysis designs with any other organizations across the
OHDSI community.</p>
<p>Tutorials for the ATLAS tool can be found
<a class="reference external" href="https://github.com/OHDSI/Atlas/wiki">here</a> and the book about the
OHDSI can be found <a class="reference external" href="https://ohdsi.github.io/TheBookOfOhdsi/">here</a>.
These resources contain a lot of useful information, particularly you
might find it useful to read about <a class="reference external" href="https://ohdsi.github.io/TheBookOfOhdsi/Cohorts.html#Cohorts">Cohort
Definition</a>.
It is highly advisable that you familiarize yourself with these
resources before proceeding. We have also provided a brief step-by-step
guide to creating dichotomous variables here:</p>
<section id="steps-to-generate-a-cohort">
<h3><strong>Steps to Generate a Cohort</strong><a class="headerlink" href="#steps-to-generate-a-cohort" title="Permalink to this heading">¶</a></h3>
<p><strong>Step 1 Create a Concept Set</strong></p>
<p>Concept Sets are a list of codes created by searching the standardized
vocabulary and selecting the set of terms for your analyses. Here you
select all concepts (variables) that you would like to use for criteria
that define your study population cohort.</p>
<p><img alt="image0" src="_images/slide_10.png" /></p>
<p>Select Concept Sets in the menu and click the blue box for “New Concept
Set”.</p>
<p><img alt="image1" src="_images/slide_11.png" /></p>
<p>In the box above the tabs, enter a name for the Concept Set you are
creating.</p>
<p>Click the green box “Add concepts” to view the list of concept
variables.</p>
<p><img alt="image2" src="_images/slide_13.png" /></p>
<p>Enter the concept name or code into the search bar to find relevant
concept variables.</p>
<p><img alt="image3" src="_images/slide_14.png" /></p>
<p>Select desired concepts, click “Add To Concept Set”. Repeat Search and
Add steps as needed, then click “Concept Sets”.</p>
<p><img alt="image4" src="_images/slide_15.png" /></p>
<p>Make sure all desired concepts are included, then click the save icon to
the right of the name of the Concept Set.</p>
<p>To add new concepts, please use the “Add Concepts” button again as
described above.</p>
<p><img alt="image5" src="_images/atlas_remove_concepts.png" /></p>
<p>To delete the concepts from the concept set, select the concepts and
click on the “Remove Selected” button, then click the save icon to save
your changes.</p>
<p><strong>Step 2 Create Cohort Definition</strong></p>
<p>A cohort is a set of persons who satisfy one or more inclusion criteria
for a duration of time.</p>
<p>Use the concept sets to create initial events and other criteria for
defining the study population cohort. A cohort is defined as a Person
with an entry and exit date. Thus, a person can be in a cohort multiple
times if they meet the criteria.</p>
<p><img alt="image6" src="_images/slide_17.png" /></p>
<p>Select “Cohort Definitions” in the menu, you can begin to create a
cohort by clicking the “New Cohort” button.</p>
<p><img alt="image7" src="_images/slide_18.png" /></p>
<p>In the boxes above and below the tabs, enter a name and description for
the Cohort.</p>
<p><img alt="image8" src="_images/slide_19.png" /></p>
<p>To establish the duration for your cohort, click “+ Add Initial Event”,
then in the dropdown menu click “Add Observation”.</p>
<p><img alt="image9" src="_images/slide_20.png" /></p>
<p>Click the blue box “Any Observation”, select “Import Concept Set” from
the dropdown menu.</p>
<p><img alt="image10" src="_images/slide_21.png" /></p>
<p>Select the Concept Set you just made. This establishes your Concept Set
as inclusion criteria for your cohort.</p>
<p><img alt="image11" src="_images/slide_22.png" /></p>
<p>To access the table in the MVP harmonization database, click “+ Add
attribute…”, then click “Add Value as Number Criteria”.</p>
<p><img alt="image12" src="_images/slide_23.png" /></p>
<p>Select “Greater or Equal To”. In this specific case we added Heart
Failure concept Set and entered value “2”. This will allow for the
collection of data from the Observation table of the MVP harmonization
database.</p>
<p>If you would like to add additional inclusion criteria, click “New
inclusion criteria” select criteria or import another configuration. To
complete the Cohort Definition, click the green Save icon.</p>
<p><strong>Step 3 Generate Cohort</strong></p>
<p>Once created, the cohort can then serve as the basis of inputs for your
subsequent analyses. Use the cohort definition to identify how many
people are in the cohort. This is the initial number that will be used
in the GWAS App when this cohort is selected.</p>
<p><img alt="image13" src="_images/slide_26.png" /></p>
<p>To complete the creation of the Cohort Definition, click “Generation”,
then “Generate”.</p>
<p>Cohort size will be displayed under the column “People”. Use View
Reports to see if you have inclusion criteria that causes cohort
attrition.</p>
<p><img alt="image14" src="_images/atlas_export_cohort.png" /></p>
<p>The “Export” tab provides a text version of how a cohort was created.
Click on the “Export” tab and then on the “Copy to clipboard” to copy
the cohort’s information.</p>
<p>We expect that this documentation in addition to the OHDSI tutorials are
sufficient for most analyses that users will attempt. If you have any
questions, please contact us at <a class="reference external" href="mailto:MVPLOI&#37;&#52;&#48;va&#46;gov">MVPLOI<span>&#64;</span>va<span>&#46;</span>gov</a>.</p>
</section>
</section>
<section id="gen3-gwas">
<h2><strong>Gen3 GWAS</strong><a class="headerlink" href="#gen3-gwas" title="Permalink to this heading">¶</a></h2>
<p>Use this App to perform a high throughput Genome-wide association
studies (GWAS) on MVP data using the University of Washington Genesis
pipeline. For more information about the Genesis pipeline please refer
to the publication <a class="reference external" href="https://doi.org/10.1093/bioinformatics/btz567">“Genetic association testing using the GENESIS
R/Bioconductor
package”</a>.</p>
<p>GWAS helps scientists identify genes associated with a particular
disease (or another trait). This method studies the entire set of DNA
(the genome) of a large group of people, searching for small variations
called single nucleotide polymorphisms or SNPs (pronounced “snips”).</p>
<p>GWAS App consists of several steps. To navigate between the steps, click
the Next or Previous box in the lower corners of the screen.</p>
<p>Each step displays the Attrition table at the top of the page. The
Attrition table contains information about the total population size and
sample size by ancestry for your selected cohorts and variables.</p>
<p>For more information about the web functionality of each step, please
refer to the button “New to GWAS? Get started here”. This tool will
offer highlighted explanations on different parts of the page.</p>
</section>
<section id="genome-wide-association-studies-gwas-steps">
<h2><strong>Genome-Wide Association Studies (GWAS) Steps</strong><a class="headerlink" href="#genome-wide-association-studies-gwas-steps" title="Permalink to this heading">¶</a></h2>
<blockquote>
<div><p><strong>Tip</strong>: On each step of the workflow there is a tutorial button
called <code class="docutils literal notranslate"><span class="pre">&quot;New</span> <span class="pre">to</span> <span class="pre">GWAS?</span> <span class="pre">Get</span> <span class="pre">started</span> <span class="pre">here&quot;</span></code>, please click the button
to navigate through the tutorial</p>
</div></blockquote>
<section id="step-1-select-study-population">
<h3><strong>Step 1 Select Study Population</strong><a class="headerlink" href="#step-1-select-study-population" title="Permalink to this heading">¶</a></h3>
<p>In this step, you will determine the study population. To begin, select
the cohort that you would like to define your study population with. You
may only see cohorts that you have access to. You may only select one
cohort. The size of the cohort population is indicated on the right-hand
side of the table. To browse the table, please scroll down to the
bottom. To search for a cohort by name you may use the search box.</p>
<p>Once a cohort has been chosen, it will populate the attrition table
above.</p>
<p>You may also see the button “Add a new cohort”. This button will open a
new tab in your browser outside of the Gen3 GWAS App and send you to the
OHDSI Atlas App, where you may create a new cohort.</p>
</section>
<section id="step-2-select-outcome-phenotype">
<h3><strong>Step 2 Select Outcome Phenotype</strong><a class="headerlink" href="#step-2-select-outcome-phenotype" title="Permalink to this heading">¶</a></h3>
<p>In this step, you will determine your outcome phenotype.</p>
<p>There are two options available: Continuous Outcome Phenotype and
Dichotomous Outcome Phenotype.</p>
<ul class="simple">
<li><p><strong>Continuous Outcome</strong></p></li>
</ul>
<p>Here, you may choose your continuous phenotype. GWAS App provides over
5,000 variables for your selection, for example, height, weight, blood
pressure, white blood cell, and other clinical observations. Please
select the outcome phenotype you wish to use in your model. All data are
harmonized from different projects through the collaborative development
of a data dictionary. To browse the table, please scroll down to the
bottom.To search the table, please enter free text in the search box to
search by phenotype name. In the plot near the table, you can learn more
about the selected outcome distribution in the study population. After
you select your outcome, click “Submit” to apply it for your model.
After clicking ‘Submit’ you may see the selection in your attrition
table.</p>
<ul class="simple">
<li><p><strong>Dichotomous Outcome</strong></p></li>
</ul>
<p>Here, you may choose your dichotomous outcome phenotype. You can define
your outcome by intersecting your study population with two other
cohorts. Please enter the name for your outcome variable in the field
“Phenotype name”. You can select your control (or No, 0) cohort in the
field “Get Value 0”, and your case cohort (or Yes, 1) in the field “Get
Value 1”. The Euler diagram on the right-hand side of the page shows the
overlap between your chosen cohorts and study population, with numbers
indicating the size of the overlap. After you define your outcome
phenotype, click “Submit” to apply it to your model. It will then
populate the attrition table and take you to the next step.</p>
</section>
<section id="step-3-select-covariate-phenotype">
<h3><strong>Step 3 Select Covariate Phenotype</strong><a class="headerlink" href="#step-3-select-covariate-phenotype" title="Permalink to this heading">¶</a></h3>
<p>This step is optional. In this step, you can add the harmonized
continuous covariates or dichotomous covariates by clicking on the
corresponding button.</p>
<ul class="simple">
<li><p><strong>Add Continuous Covariate</strong></p></li>
</ul>
<p>GWAS App provides over 5,000 variables for your selection, for example,
height, weight, blood pressure, white blood cell, and other clinical
observations. Please select all covariates you wish to use in your
model. You may choose as many covariates as you wish in this step. To
browse the table, please scroll down to the bottom.To search the table,
please enter free text in the search box to search by the covariate
name. You can learn more about the selected covariate distribution in
the study population in the plot near the table. After you select your
covariate, click “Add” to apply it to your model. The covariate will
then appear on the right-hand side of the screen and populate the
attrition table. You can delete previously chosen covariates by clicking
on the Delete icon.</p>
<ul class="simple">
<li><p><strong>Add Dichotomous Covariate</strong></p></li>
</ul>
<p>You may add custom dichotomous covariates by selecting two cohorts.
Please enter the name for your covariate variable in the field
“Covariate name”. You can select your control (or No, 0) cohort in the
field “Get Value 0” and your case cohort (or Yes, 1) in the field “Get
Value 1”. The Euler diagram on the right-hand side of the page shows the
overlap between your chosen cohorts and study population with numbers
indicating the size of the overlap. After you define your covariate,
click “Add” to apply it to your model. The covariate will then appear on
the right-hand side of the screen and populate the attrition table. You
can delete previously created covariates by clicking on the Delete icon.</p>
</section>
<section id="step-4-configure-gwas">
<h3><strong>Step 4 Configure GWAS</strong><a class="headerlink" href="#step-4-configure-gwas" title="Permalink to this heading">¶</a></h3>
<p>In this step, you will determine workflow parameters. Please adjust the
number of population principal components (PCs) to control for
population structure, minor allele frequency cutoff, and imputation
score cutoff. Please also choose one ancestry population on which you
would like to perform your study.</p>
<ul class="simple">
<li><p><strong>Number of PCs</strong> - Population Principal components (PCs) refer to
linear combinations of genome-wide genotyping data to control for
population structure/stratification (select up to 10 PCs).</p></li>
<li><p><strong>Harmonized Ancestry and Race/Ethnicity (HARE) dropdown menu</strong> -
Please choose the ancestry population on which you would like to
perform your study. The numbers appearing in the dropdown represent
the population size of your study, considering all of your previous
selections. You may not proceed with the analysis without a
selection.</p></li>
<li><p><strong>MAF Cutoff- Minor allele frequency (MAF)</strong> is the frequency at
which the second most common allele occurs in a given population and
can be used to filter out rare markers (scale of 0-0.5).</p></li>
<li><p><strong>Imputation Score Cutoff</strong> - This value reflects the quality of
imputed SNPs and can be used to remove low-quality imputed markers
(scale of 0-1). Once you have selected your values please press Next.</p></li>
</ul>
</section>
<section id="submission-modal-and-additional-options">
<h3><strong>Submission Modal and Additional Options</strong><a class="headerlink" href="#submission-modal-and-additional-options" title="Permalink to this heading">¶</a></h3>
<p>Once step 4 is concluded, you may review your details in a modal. This
will present to you the configurable selections made in every step of
the GWAS flow.</p>
<ul class="simple">
<li><p><strong>Configurable values</strong> - May be changed in step 4.</p></li>
<li><p><strong>Covariates</strong> - Please review the chosen covariates. To remove
unwanted covariates, or change them, please go back (at the bottom of
the page) to step 3.</p></li>
<li><p><strong>Outcome Phenotype</strong> - Please see here the outcome phenotype chosen
in step 2. To remake the selection please go back (at the bottom of
the page) to step 2.</p></li>
<li><p><strong>Cohort</strong> - This represents the study population you selected. To
change your selection please go back to step 1.</p></li>
</ul>
<p>Once you have reviewed your selections you must give a name to your
analysis, followed by pressing ‘Submit’.</p>
<p>Once the job is submitted, a success message will then appear on the
screen with the given job name and GWAS job id. You may progress in
three different paths-</p>
<ol class="arabic simple">
<li><p>See status - This option will send you to a different page where you
may review the status of your job.</p></li>
<li><p>Submit new workflow - This option will send you to the beginning of
the GWAS App where you may submit a different GWAS for analysis.</p></li>
<li><p>Submit similar (stay here) - This option will keep you at this page,
where you may be able to submit some changes to the GWAS you already
built and submit a similar job.</p></li>
</ol>
</section>
<section id="check-submission-status-and-review-results">
<h3><strong>Check Submission Status and Review Results</strong><a class="headerlink" href="#check-submission-status-and-review-results" title="Permalink to this heading">¶</a></h3>
<p>Once your GWAS analysis is submitted, you can check the Submission
Status and Review the Results in the “GWAS Results” App.</p>
</section>
</section>
<section id="gwas-results">
<h2><strong>GWAS Results</strong><a class="headerlink" href="#gwas-results" title="Permalink to this heading">¶</a></h2>
<p>Use this App to view the status &amp; results of submitted workflows. Click
the arrow in the Submitted Job Statuses box to activate the drop down
menu and see the status of your analysis. This menu will display a
history of your submitted jobs including the Run ID of your analysis,
the start time, and the finish time when the run is completed. The menu
will also show the status of your analysis:</p>
<ul class="simple">
<li><p>Pending - your analysis was placed in the queue to run, depending on
the length of the queue your analysis could start in the range of
several minutes to several hours</p></li>
<li><p>In Progress - your analysis is started and running, depending on your
selection of cohort and variables it could finish in the range of
half an hour to three hours</p></li>
<li><p>Failed - your analysis returned the error, it is advised to
review/change your GWAS parameters or contact our help desk at
<a class="reference external" href="mailto:MVPLOI&#37;&#52;&#48;va&#46;gov">MVPLOI<span>&#64;</span>va<span>&#46;</span>gov</a></p></li>
<li><p>Completed - your analysis was successfully completed and you may
download the results of the GWAS analysis from this menu</p></li>
</ul>
<p>Once completed, you may download the results of the GWAS analysis from
this menu. By pressing the ‘Download’ link a zip file will start
downloading to your computer. Depending on your cohort and variables
selection, the file size can vary in the range of 700MB up to 1.3GB. The
file contains the following: Manhattan plot, QQ plot, metadata file
containing all of your selections, your study’s attrition table, and
per-chromosome GWAS summary statistics.</p>
</section>
</section>
<section id="frequently-asked-questions">
<h1>Frequently Asked Questions<a class="headerlink" href="#frequently-asked-questions" title="Permalink to this heading">¶</a></h1>
<section id="where-can-i-find-more-documentation-on-the-ohdsi-atlas-app">
<h2><strong>Where can I find more documentation on the OHDSI Atlas app?</strong><a class="headerlink" href="#where-can-i-find-more-documentation-on-the-ohdsi-atlas-app" title="Permalink to this heading">¶</a></h2>
<p>ATLAS was developed and is maintained by the OHDSI community. Tutorials
for the ATLAS tool can be found
<a class="reference external" href="https://github.com/OHDSI/Atlas/wiki">here</a> and the book about the
OHDSI can be found <a class="reference external" href="https://ohdsi.github.io/TheBookOfOhdsi/">here</a>.
These resources contain a lot of useful information, particularly you
might find it useful to read about <a class="reference external" href="https://ohdsi.github.io/TheBookOfOhdsi/Cohorts.html#Cohorts">Cohort
Definition</a>.
If you need help, please reach out to our help desk at <a class="reference external" href="mailto:MVPLOI&#37;&#52;&#48;va&#46;gov">MVPLOI<span>&#64;</span>va<span>&#46;</span>gov</a></p>
</section>
<section id="what-are-harmonized-variables">
<h2><strong>What are harmonized variables?</strong><a class="headerlink" href="#what-are-harmonized-variables" title="Permalink to this heading">¶</a></h2>
<p>Data harmonization is the process of defining terms across different but
similar projects. It helps to avoid inconsistencies in data reporting
and makes it easier to find and analyze data. For example, different
studies can have their unique names for the variable “date a participant
enrolled”, like “index date”, “date of enrollment”, or “AnchorDate”.
Those variables are harmonized, or mapped, to a single harmonized
variable “enrollment date”.</p>
</section>
<section id="what-will-be-included-in-the-downloadable-package-at-the-end-of-a-gwas">
<h2><strong>What will be included in the downloadable package at the end of a GWAS?</strong><a class="headerlink" href="#what-will-be-included-in-the-downloadable-package-at-the-end-of-a-gwas" title="Permalink to this heading">¶</a></h2>
<p>The package contains the following: Manhattan plot, QQ plot, metadata
file containing all of your selections, your study’s attrition table,
and per-chromosome GWAS summary statistics.</p>
</section>
<section id="how-long-will-my-gwas-take">
<h2><strong>How long will my GWAS take?</strong><a class="headerlink" href="#how-long-will-my-gwas-take" title="Permalink to this heading">¶</a></h2>
<p>You may check the status of your analysis in the “GWAS Results” App.
After you submit your analysis, it will be placed in the queue to run.
The “GWAS Results” App will show the “Pending” status for your submitted
analysis. Depending on the length of the queue, your analysis could
start in the range of several minutes to several hours. After the
analysis starts to run, the status will be changed to “In Progress”.
Depending on your selection of cohort and variables, it could finish in
the range of half an hour to three hours. You may close your browser
after you submit the analysis.</p>
</section>
<section id="why-does-the-workspaces-page-give-me-an-error">
<h2><strong>Why does the workspaces page give me an error?</strong><a class="headerlink" href="#why-does-the-workspaces-page-give-me-an-error" title="Permalink to this heading">¶</a></h2>
<p>Currently, workspaces are not available, and therefore you may see an
error when attempting to log into this page. Generally, workspaces are
secure data analysis environments in the cloud that can access data from
one or more data resources. Workspaces may include Jupyter notebooks and
JupyterLab, Python and RStudio. For more information about the Gen3
Workspace, you may refer to <a class="reference external" href="https://gen3.org/products/workspaces/">Gen3
Workspaces</a> and <a class="reference external" href="https://gen3.org/resources/user/analyze-data/">Data Analysis
in a Gen3 Data
Commons</a>.</p>
</section>
</section>


          </div>
          
        </div>
      </div>
      <div class="sphinxsidebar" role="navigation" aria-label="main navigation">
        <div class="sphinxsidebarwrapper">
<h1 class="logo"><a href="#">va-doc</a></h1>








<h3>Navigation</h3>

<div class="relations">
<h3>Related Topics</h3>
<ul>
  <li><a href="#">Documentation overview</a><ul>
  </ul></li>
</ul>
</div>
<div id="searchbox" style="display: none" role="search">
  <h3 id="searchlabel">Quick search</h3>
    <div class="searchformwrapper">
    <form class="search" action="search.html" method="get">
      <input type="text" name="q" aria-labelledby="searchlabel" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false"/>
      <input type="submit" value="Go" />
    </form>
    </div>
</div>
<script>document.getElementById('searchbox').style.display = "block"</script>








        </div>
      </div>
      <div class="clearer"></div>
    </div>
    <div class="footer">
      &copy;2022, phyllis pischl, zuyi chen.
      
      |
      Powered by <a href="http://sphinx-doc.org/">Sphinx 6.1.3</a>
      &amp; <a href="https://github.com/bitprophet/alabaster">Alabaster 0.7.13</a>
      
      |
      <a href="_sources/index.rst.txt"
          rel="nofollow">Page source</a>
    </div>

    

    
  </body>
</html>