<!DOCTYPE html>
<html>

    <head>
        <meta http-equiv='Content-Type' content='text/html; charset=UTF-8'> 
        <link rel='stylesheet' type='text/css' href='./index.css' />
        <link rel="preconnect" href="https://fonts.gstatic.com">
        <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@200;300;400&display=swap" rel="stylesheet">
        <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
        <link rel='icon' type='image/x-icon' />
        <title>Documentation</title>
    </head>
    <body>
        <div class="arrow">
            <a href="#" title="Back to Top"><span class="material-icons">arrow_upward</span></a>
          </div>
      

    <div class="sidenav">
        <strong>><a href="#HostedDataOverview">Hosted Data Overview</a></strong>
        <strong><a href="#HowToGetStarted">How to get started</a></strong>
        <ul>
           
        <a href="#Login">1. Login to the ACCOuNT Data Commons</a>
        <a href="#Explorer">2. Data Explorer</a>
        <a href="#Download">3. Downloading Data and Data Files</a>
        <a href="#Datamodel">4. Understanding the Gen3 data model</a>
    
        </ul>

    </div>
      

         

    <div id="root">
        <div class='top-bar'>
            <div class='top-bar__header'>
            </div>
        </div>
        <div class='nav-bar'>
            <div class='nav-bar__header'>
                <div class='nav-bar__logo'>
                    <img src='img/header-logo.png' class='nav-bar__logo-img' />
                </div>
            </div>
        </div>
        <div class='main-content'>
            <div class='section-title'>
                <h1>User Guide for ACCOuNT Data Commons</h1>
            </div>
            <div class='other-info'>
            <h2 id="HostedDataOverview" class="section-title">Hosted Data Overview</h2>
            <p>ACCOuNT currently hosts genotyping, sequencing, and clinical data from the following programs:</p>
            <p> 
                <ul>
                    <li>
                        Open Access (Open-GEO)
                    </li>
                    <li>
                        NU African American Hepatocytes
                    </li>
                    <li>
                        The Discovery Project
                    </li>
                    <li>
                        MESA (Multi-Ethnic Study of Atherosclerosis)
                    </li>
                </ul>
            </p>
            <p>
                We list a description of each program and associated studies below.
            </p>
                
            <p>
                <span style="font-weight: bold;">Open-GEO</span> contains genotype data for more than 300 participant samples 
                associated with lung cancer cell lines, as well as associated phenotype 
                information. The genotype data consists of mRNA microarrays and 
                unaligned reads in the form of CEL and FASTQ files. Phenotype data 
                includes information related to demographics, exposure, diagnosis, 
                treatment, follow-up, and medical history of a subject. For more 
                information on GEO, please visit the NCBI homepage located 
                <a href="https://www.ncbi.nlm.nih.gov/geo/">here</a>.
            </p>
            <p>
                <span style="font-weight: bold;">NU African American Hepatocyte:</span> 
                This study contains genotype and associated transcriptome profile 
                from hepatocytes obtained from African American livers. Transcriptomes 
                were measured under the following seven treatments: Phenobarbital, 
                Dexamethasone, Carbamazepine, Phenytoin, Rifampicin, DSMO, and 
                Omeprazole. Indications on which drug is used can be found on the 
                last digit of the submitter_id:
                XX-YY-Z, where Z displays a number from 1 to 7 indicating which 
                drug is used. DSMO is 1, Omeprazole is 2, Phenobarbital is 3, 
                Dexamethasone is 4, Carbamazepine is 5, Phenytoin is 6, and 
                Rifampicin is 7.
                Genotype data includes submitted aligned reads and aligned read 
                index files in the form of BAM files. Phenotype data consists of 
                basic demographics when available.  All data is also available 
                at <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147628">GSE147628</a>, 
                <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE124074">124074</a>, and 
                <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123995">123995</a>.
                
                Note: “NU” stands for “Northwestern University”.
            </p>
            <p>
                <span style="font-weight: bold;">The Discovery Project</span> identifies genetic biomarkers (or SNPs)
                to cardiovascular drug response and disease susceptibility. 
                “The Discovery Project” currently has phenotype and genotype 
                data stored across 4 projects or research “arms.” All projects, 
                except for healthy_platelet, contain demographic, drug phenotypic 
                and clinical data. The particular elements for each arm are listed 
                below. All arms have basic demographic data. Below we describe 
                the data listed for each project:
            </p>
                <ul>
                    <li>
                        <span style="font-weight: bold;">Healthy_platelet:</span> Genotype files consist of unaligned reads, 
                        SNP arrays, genotyping arrays, simple germline variations, 
                        germline variation indices in the form of FASTQ, CEL, IDAT, 
                        VCF, and CSI files. Transcriptome data is available as FASTQ 
                        and BAM files. These are healthy African American volunteers with 
                        age values (sex is also available).

                    </li>
                </ul>
                <ul>
                    <li>
                        <span style="font-weight: bold;">Warfarin:</span> Genotype files 
                        including simple germline variations, germline variation 
                        indices, and genotyping arrays in the form of IDAT, VCF, 
                        and CSI files. In addition, this study contains a CSV file 
                        with a comprehensive clinical dataset. Example clinical 
                        properties in this file include but are not limited to a 
                        list of comorbidities, warfarin dose, target INR, reason 
                        for warfarin treatment, and adverse bleeding events that 
                        occured while on treatment. All subjects were followed for 
                        6-months after enrollment in the study.
                    </li>
                </ul>
                <ul>
                    <li>
                        <span style="font-weight: bold;">Novel Oral Anticoagulants (NOAC):</span> 
                        Genotype files for simple germline variations, germline variation 
                        indices, and genotyping arrays in the form of IDAT, VCF, and 
                        CSI files. In addition, this study contains a CSV file with 
                        a comprehensive clinical dataset. Example clinical properties 
                        in this file include but are not limited: list of comorbidities, 
                        which NOAC drug was used, therapeutic dose, or list of active
                         medications (RX and OTC) and adverse bleeding and clotting 
                         events that occured while on treatment, and a clinical Anti-Xa 
                         lab measure taken while on treatment. All subjects were 
                         followed for 6-months after enrollment in the study.
                    </li>
                </ul>
                <ul>
                    <li>
                        <span style="font-weight: bold;">Clopidogrel:</span> 
                        Genotype files that include simple germline variations, germline 
                        variation indices, genotyping arrays, and SNP arrays in the form 
                        of IDAT, CEL, VCF, and CSI files. In addition, this study contains a
                         CSV file with a comprehensive clinical dataset. Example clinical 
                         properties in this file include but are not limited to the 
                         following properties: list of comorbidities, clopidogrel dose, 
                         aspirin/NSAIDs dose, proton pump inhibitors, other medications, 
                         adverse bleeding and clotting events that occured while on treatment,
                          and a clinical PRU lab measure taken while on treatment. 
                          All subjects were followed for 6-months after enrollment in the study.        
                    </li>

                </ul>
            <p>
                <span style="font-weight: bold;">MESA</span> (Multi-Ethnic Study of 
                Atherosclerosis) contains -omics data, such as genomic, transcriptomic, 
                and proteomic data. This program consists of 2 studies with data files
                including ZIP, TAR, LOG, and TXT. The 2 available studies are: 
                <ul>
                    <li>
                        MESA_AACAC
                    </li>
                    <li>
                        SHARe
                    </li>
                </ul>
                Source: A link to the NIH MESA page can be found <a href="https://www.nhlbi.nih.gov/science/multi-ethnic-study-atherosclerosis-mesa">here</a>.
            </p>
  
            <figure>
                <img src="img/1-Subject-File-Overlook.png" alt="Subject-File-Overlook" style="width:70%">
                <figcaption>
                    Fig. 1. The list of projects on the ACCOuNT data commons in Gen3 in 
                    the “Subject” Tab (left) and “File” Tab (right). Numbers represent 
                    the numbers of subjects (left) or files (right) associated with the 
                    project (e.g. warfarin). Caution -- the MESA studies are only 
                    displayed on the “File Tab”.
                </figcaption>
            </figure>
            <h2 id="HowToGetStarted" class="section-title">How to get started</h2>
            

            <h3 id="Login" class="section-title">1. Login to the ACCOuNT Data Commons</h3>
            <p style="text-align: justify; line-height: 1.5;">
            <h4>
                a. Homepage
            </h4>
            <p>
                In order to navigate and access data available on the Gen3 
                platform, please start by visiting the login page 
                (<a href="https://acct.bionimbus.org">https://acct.bionimbus.org</a>). Use the “Login to Google” 
                on the homepage to authenticate:
            </p>    
            <figure>
                <img src="img/2-Account-Login-Page.png" alt="Account-Login-Page" style="width:70%">
                <figcaption>
                    Fig. 2. Login Page of ACCOuNT.
                </figcaption>
            </figure>
            </span>
            <p>
                After successfully logging in, your username will appear 
                in the top-right corner of the page. You will also see 
                a display with summary statistics for the total number of 
                projects, subjects, and files available within the ACCOuNT 
                Data Commons.
            </p>
            <figure>
                <img src="img/3-Account-Post-Login-Page.png" alt="Account-Post-Login-Page" style="width:70%">
                <figcaption>
                    Fig. 3. Post-login view of the ACCOuNT front page.
                </figcaption>
            </figure>

            <h4>
                b. How to check your data access
            </h4>
            <p>
                To check your user access information, go to the website 
                (<a href="https://acct.bionimbus.org">https://acct.bionimbus.org</a>) and click on “Login from Google”. 
                In the “Exploration” tab located at the right-hand side of the 
                page, you will be presented with the data you have access to under 
                the Studies Sub-tab, Project_Id.  
            </p>
            <p>
                Similarly, you can visit the “Profile” page and view a 
                list of your approved studies under the “You have access to 
                the following project(s)” section, as shown in the figure below.
            </p>
            <figure>
                <img src="img/4-Account-Profile-Page.png" alt="Account-Profile-Page" style="width:70%">
                <figcaption>
                    Fig. 4. A user can view their study access in the Profile page.
                </figcaption>
            </figure>
            <p>
                Currently, the ACCOuNT Data Commons hosts only 1 open access dataset, 
                <span style="font-weight: bold;">open-GEO</span>. Users can request access to controlled data by 
                visiting the dbGaP homepage or get into contact with the 
                ACCOuNT consortium (<a href="https://precisionmedicine4all.com">https://precisionmedicine4all.com</a>).
            </p>

            
            

            <h3 id="Explorer" class="section-title">2. Data Explorer</h3>
            <p style="text-align: justify">
                <a href="https://acct.bionimbus.org/explorer">The Exploration Page</a> 
                provides users with a venue to easily 
                search through data and create cohorts via faceted search 
                fields. By leveraging the different faceted search categories, 
                users can not only create virtual cohorts within individual studies, 
                but also across any of the available studies (with proper user 
                authorization). Below, we explore the different core sections of 
                the Exploration page.
            </span>

            <h4>
                a. The Subject Tab
            </h4>
            <p>
                Under the “Subject” Tab, users can leverage the ACCOuNT data 
                model and a list of available phenotypic properties to create 
                virtual cohorts. The selection of search facets will dynamically 
                update the virtual cohort at the subject level. If no facets have 
                been selected, all of the data accessible to the user will be 
                displayed. At this time, users can filter based on properties 
                grouped into 3 categories displayed as sub-tabs: Studies, 
                Demographics and Clinical (image below).
            </p>
            <figure>
                <img src="img/5-Exploration-page-with-Access.png" alt="Exploration-page-with-Access" style="width:70%">
                <figcaption>
                    Fig. 5. Exploration page viewed as a user with “Data Access”. 
                    The Subject Tab on the Exploration page hosts ACCOuNT-custom tabs, 
                    filters, export buttons, and summary statistics.
                </figcaption>
            </figure>
            <figure>
                <img src="img/6-Exploration-page-with-details.png" alt="Exploration-page-with-details" style="width:70%">
                <figcaption>
                    Fig. 6. Details for custom specs found on the Subject Tab in the Exploration 
                    page. The Subject Tab can be used to create custom cohorts, export to workspaces 
                    or download data to the local environment.
                </figcaption>
            </figure>

            <h5>
                Open Access Data
            </h5>
            <p>
                A user can view all of the summary data and associated study information they 
                have access to, including but not limited to Project ID, file types, and clinical 
                variables. Currently, the ACCOuNT Data Commons hosts open access data seen in the 
                project <span style="font-weight: bold;">open-GEO</span>.
            </p>
            <figure>
                <img src="img/7-View-summary-information.png" alt="View-summary-information" style="width:70%">
                <figcaption>
                    Fig. 7. Users may only view summary information for projects with open access.
                </figcaption>
            </figure>
            <p>
                Users can request access by visiting the <a href="https://dbgap.ncbi.nlm.nih.gov/aa/wga.cgi?page=login">dbGaP webpage</a> 
                or get into contact with the <a href="https://precisionmedicine4all.com">ACCOuNT consortium</a>.
            </p>
            <h4>
                b. The File Tab
            </h4>
            <p>
                The File tab provides users with search facets in order to dynamically generate lists 
                of desired files  (“Projects”, “File Category”, “File Type”, etc.). The complete list 
                of selected files and associated metadata will be displayed in the table at the bottom 
                right-hand corner of the page.
            </p>
            <figure>
                <img src="img/8-File-Tab.png" alt="File-Tab" style="width:70%">
                <figcaption>
                    Fig. 8. The File tab on the Exploration page shows data files on the ACCOuNT Data 
                    Commons. The File Tab can be used to create custom cohorts, export to workspaces or 
                    download data files to the local environment.
                </figcaption>
            </figure>





           
            <h3 id="Download" class="section-title">3. Downloading Data and Data Files</h3>
            <p style="text-align: justify; line-height: 1.5;">
                Users can download clinical metadata directly from the Data Explorer (section 
                a below) or download files to their local environment by following 2 different 
                processes. The first process allows for single-file downloads directly through 
                the portal, and is explained in Section b below. The second process allows for 
                multi-file downloads by leveraging the Gen3-client and a python SDK, as 
                detailed in Section c.
            </p>
            <h4>
                a. Clinical Data and File Metadata Downloads via the Subject and File Tab
            </h4>
            <p>
                Users can download clinical data and file metadata from the Explorer Page. 
                On the Subject Tab, users can download dynamically updated virtual cohorts at the subject level in table format, for example demographics, clinical data, and the submitter_ids. 
                On the File Tab, users can download metadata associated with the data files, such as size, guids, data type, and submitter_ids.
            </p>
            <p>
                Users need to click “Download Table” to save it to the local machine. 
            </p>
            <figure>
                <img src="img/9-Metadata-and-clinical-data-download.png" alt="Metadata-and-clinical-data-download" style="width:70%">
                <figcaption>
                    Fig. 9. Download clinical data at subject-level from the Subject Tab (top) and 
                    file metadata from the File Tab (bottom).
                </figcaption>
            </figure>



            <h4>
                b. Single-file Downloads via the File Tab
            </h4>
            <p>
                Users can download files individually by visiting the summary table in the 
                File tab (Refer to Section 2b for more information). The table provides a column 
                titled GUID (<i>Globally Unique Identifier</i>) with a link to a download page.
            </p>
            <figure>
                <img src="img/10-Exploration-Table.png" alt="Exploration-Table" style="width:70%">
                <figcaption>
                    Fig. 10. Each file row includes a GUID column that provides a link to a download page.
                </figcaption>
            </figure>
            <p>
                Once a GUID is selected, the user will be redirected to the download page as displayed below: 
            </p>
            <figure>
                <img src="img/11-Download-Data-File.png" alt="Download-Data-File" style="width:70%">
                <figcaption>
                    Fig. 11. The Download button will start the download process for that file to your local directory.
                </figcaption>
            </figure>


            <h5>
                Example: Finding and Downloading Individual Phenotypic Data Files
            </h5>
            <figure>
                <img src="img/Download-Data-File-GIF.gif" alt="Download-Data-File-GIF" style="width:70%">
            </figure>  
            <ol>
                    <li>
                        Under the File Tab, select “CSV” beneath the “File Format” facet.
                    </li>
                    <li>
                        Select one of the GUIDs (<i>Globally Unique Identifier</i>) stored for the phenotypic data files.
                    </li>
                    <li>
                        Click “Download” on the next page.
                    </li>
            </ol>
            
            <h4>
                b. Downloading Multiple Data Files
            </h4>
            <p>
                In order to download multiple files, you will need to utilize the gen3-client command 
                line tool developed by the University of Chicago’s Center for Translational Data Science. 
                For detailed information, you can visit the following page - <a href="https://gen3.org/resources/user/gen3-client/">https://gen3.org/resources/user/gen3-client/</a>. 
                The process consists of the 4 following steps:
            </p>
            <ol>
                <li>
                    Create and download an API key.
                </li>
                <li>
                    Prepare and download a file manifest of your selected cohort.
                </li>
                <li>
                    Download and configure the gen3-client.
                </li>
                <li>
                    Download the files.
                </li>
            </ol>

            <h5>
                <span style="font-weight: bold;">1. Create and save an API key from the “Profile” page:</span>
            </h5>

            <figure>
                <img src="img/12-Create-API-Key.png" alt="Create-API-Key" style="width:70%">
                <figcaption>
                    Fig. 12. Create API key
                </figcaption>
            </figure>
            <figure>
                <img src="img/13-Download-API-Key.png" alt="Download-API-Key" style="width:70%">
                <figcaption>
                    Fig. 13. Download API key and note the directory where it was saved. 
                </figcaption>
            </figure>

            <h5>
                <span style="font-weight: bold;">2. Prepare and download a file manifest </span>
            </h5>
            <ol type="a">
                <li>
                    Under the File Tab, select all data files of interest. Users can sort 
                    after project, file type, file category, file format, or find file names 
                    in the free text search bar.
                </li>
                <li>
                    Click on the “Download Manifest” button. Note the directory where it was saved. 
                </li>
            </ol>
            <figure>
                <img src="img/Prepare-file-manifest-GIF.gif" alt="Prepare-file-manifest-GIF" style="width:70%">
                <figcaption>
                    Fig. 14. Prepare and download a file manifest. 
                </figcaption>
            </figure>

            <h5>
                <span style="font-weight: bold;">3. Download and configure the Gen3-client</span>
            </h5>
            <ol type="a">
                <li>
                    Follow the download instructions of the Gen3-client <a href="https://gen3.org/resources/user/gen3-client/#1-installation-instructions">here</a>.
                </li>
                <li>
                    In your terminal, configure your profile using the following command:
                </li>
            </ol>
            <code style="width:70%">
                gen3-client configure --profile=&lt;profile_name&gt; --cred=&lt;credentials.json&gt; --apiendpoint=&lt;api_endpoint_url&gt;
            
            </code>
            <p>
                Example on Mac/Linux:
            </p>
            <code>
                gen3-client configure --profile=demo --cred=~/Downloads/demo-credentials.json --apiendpoint=https://acct.bionimbus.org/
            </code>
            <p>
                Example on Windows:
            </p>
            <code>
                gen3-client configure --profile=demo --cred=C:\Users\demo\Downloads\demo-credentials.json --apiendpoint=https://acct.bionimbus.org/
            </code>
<pre style="width:70%">
2021/02/23 10:08:20 Profile 'demo' has been configured successfully.
</pre>
            <p>
                If successfully executed, a configuration file will be stored under the directory the user specified under “cred”.
            </p>
            <p>
                Users should confirm if the profile was configured successfully using the following command, which should list your 
                access privileges for each project in the commons you have access to:
            </p>

            <code>
                gen3-client auth --profile=demo
            </code>
<pre style="width:70%">
2021/02/23 10:12:10 
You have access to the following project(s) at https://acct.bionimbus.org:
2021/02/23 10:12:10 discovery [create delete read read-storage update upload]
2021/02/23 10:12:10 mesa [create delete read read-storage update upload]
2021/02/23 10:12:10 nu [create delete read read-storage update upload]
2021/02/23 10:12:10 open [create delete read read-storage update upload]
</pre>
            <p>
                For troubleshooting, refer to the instructions found <a href="https://gen3.org/resources/user/gen3-client/#2-configure-a-profile-with-credentials">here</a>. 
            </p>


            <h5>
                <span style="font-weight: bold;">4. Download the data with the file manifest generated in Step 2</span>
            </h5>
            <p>
                Run the following commands in the terminal:
            </p>
            <code>
                gen3-client download-multiple --profile=&lt;profile_name&gt; --manifest=&lt;manifest_file&gt; --download-path=&lt;path_for_files&gt;
            </code>
        <p>
            Example:
        </p>
        <code>
            gen3-client download-multiple --profile=demo --manifest=manifest.json --download-path=downloads
        </code>
<pre style="width:70%">
2021/02/23 10:26:58 Reading manifest...
598 B / 598 B [======] 100.00% 0s
2021/02/23 10:27:00 Total number of GUIDs: 3
2021/02/23 10:27:00 Preparing file info for each file, please wait...
3 / 3 [======] 100.00% 0s
2021/02/23 10:27:00 File info prepared successfully
account_warfarin_processed.csv  243.71 KiB / 243.71 KiB [======] 100.00%
account_clopidogrel_processed.csv  197.93 KiB / 197.93 KiB [======] 100.00%
account_noac_processed.csv  184.89 KiB / 184.89 KiB [======] 100.00%
2021/02/23 10:27:02 3 files downloaded.
</pre>
        <p>
            For more information see the <a href="https://gen3.org/resources/user/gen3-client/#5-provide-a-manifest-file-for-bulk-download">User Guide</a>.
        </p>



            <h3 id="Datamodel" class="section-title">4. Understanding the Gen3 data model</h3>
            <p style="text-align: justify; line-height: 1.5;">
                Gen3-powered Data Commons employ a data model which describes, organizes, 
                and establishes relationships for data across different datasets. The data 
                model organizes and connects different categories, <i>"nodes"</i>, with experimental 
                metadata variables, <i>“properties”</i>. The data dictionary describes all nodes and 
                enlisting properties in each node. 
            </p>
            <p>
                On ACCOuNT, the dictionary page contains an interactive representation of the 
                Gen3 data model in two forms, “graph model” and “table view”, which is described below. 
            </p>
            <p>
                <em>Note:</em> the Gen3 Data Dictionary encompasses <i>all study metadata</i> displayed 
                in the Gen3 ACCOuNT portal. Not all studies will contain data associated with 
                the different nodes and properties available.
            </p>
            <figure>
                <img src="img/Gen3-Dictionary-GIF.gif" alt="Gen3-Dictionary-GIF" style="width:70%">
                <figcaption>
                    Fig. 14. Navigating the Data Dictionary on the ACCOuNT (<a href="https://acct.bionimbus.org/dd">https://acct.bionimbus.org/dd</a>) in “Table” or “Graph” view. 
                </figcaption>
            </figure>
            <p>
                Node types, categories, and relationships between nodes are displayed in a 
                hierarchical view. Links and node categories are highlighted in the legend 
                (top right-hand side). Click on nodes to see properties and download available 
                templates (“JSON”, “TSV”). Use the dictionary search bar (top left-hand side) to 
                search through property names and descriptions. For more information on the Gen3 
                data model see <a href="https://gen3.org/resources/user/dictionary">https://gen3.org/resources/user/dictionary/</a>. 
            </p>


            </div>



            
                </div>
        </div>
        <div class='footer'>
            <div class="footer__logo-area">
                <img class='footer__img' src='img/gen3.png' />
                <img class='footer__img' src='img/createdby.png' />
            </div>
        </div>
    </div>

    <script>
        function readMore() {
            var dots = document.getElementById("dots");
            var moreText = document.getElementById("more");
            var btnText = document.getElementById("read-more-btn");
            var btnText = document.getElementById("read-more-explorer");

            if (dots.style.display === "none") {
                dots.style.display = "inline";
                btnText.innerHTML = "Read more";
                moreText.style.display = "none";
            } else {
                dots.style.display = "none";
                btnText.innerHTML = "Read less";
                moreText.style.display = "inline";
            }
        }
    </script>






</body>

</html>