<html>
<head>
  <title>Gen3 Open Links</title>

  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <meta http-equiv="Content-Security-Policy"
    content="default-src 'none';
    img-src 'self' data: https://www.google-analytics.com;
    script-src 'self' https://www.google-analytics.com;
    style-src 'self' https://fonts.googleapis.com;
    object-src 'none';
    worker-src 'self';
    connect-src 'self';
    font-src 'self' https://fonts.googleapis.com https://fonts.gstatic.com"
    >
  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />

  <link href="./app.css" rel="stylesheet" />

</head>
<body>
  <header>
    <div class="row">
      <div class="logo-row">
        <img src="./blood-pac.img" alt="logo" class="logo"></img>
      </div>
    </div>
  </header>
  <h1>Treatment response and tumor evolution: lessons from an extended series of multianalyte liquid biopsies in a metastatic breast cancer patient<h1>
  <h2>Abstract</h2>

  <p>Currently, clinical characterization of metastatic breast cancer is based on tissue samples taken at time of diagnosis. However, tissue biopsies are invasive and tumors are continuously evolving, which indicates the need for minimally invasive longitudinal assessment of the tumor. Blood-based liquid biopsies provide minimal invasive means for serial sampling over the course of treatment and the opportunity to adjust therapies based on molecular markers. Here, we aim to identify cellular changes that occur in breast cancer over the lifespan of an affected patient through single-cell proteomic and genomic analysis of longitudinally sampled solid and liquid biopsies. Three solid and 17 liquid biopsies from peripheral blood of an ER+ /HER2âˆ’ metastatic breast cancer patient collected over 4 yr and eight treatment regimens were analyzed for morphology, protein expression, copy-number alterations, and single-nucleotide variations. Analysis of 563 single morphometrically similar circulating tumor cells (CTCs) and 13 cell-free DNA (cfDNA) samples along with biopsies of the primary and metastatic tumor revealed progressive genomic evolution away from the primary tumor profiles, along with changes in ER expression and the appearance of resistance mutations. Both the abundance and the genomic alterations of CTCs and cfDNA were highly correlated and consistent with genomic alterations in the tissue samples. We demonstrate that genomic evolution and acquisition of drug resistance can be detected in real time and at single-cell resolution through liquid biopsy analytes and highlight the utility of liquid biopsies to guide treatment decisions.</p>

  <table class="dcf-table dcf-table-responsive dcf-table-bordered dcf-table-striped dcf-w-100%">
	<tbody>
		<tr>
			<td>Authors</td>
			<td><span>Lisa Welter<sup>1</sup>, Liya Xu<sup>1</sup>, Dillon McKinley<sup>1</sup>, Angel E. Dago<sup>2</sup>, Rishvanth K. Prabakar<sup>1</sup>, Sara Restrepo-Vassalli<sup>1</sup>, Kevin Xu<sup>1</sup>, Mariam Rodriguez-Lee<sup>1</sup>, Anand Kolatkar<sup>1</sup>, Rafael Nevarez<sup>1</sup>, Carmen Ruiz<sup>1</sup>, Jorge Nieva<sup>3</sup>, Peter Kuhn<sup>1,3,4</sup>, and James Hicks<sup>1</sup></span></td>
		</tr>
		<tr>
			<td>Affiliations</td>
			<td><span>1. Convergent Science Institute in Cancer, Michelson Center for Convergent Bioscience, University of Southern California, Los Angeles, California 90089, USA,<br> 2. The Scripps Research Institute, La Jolla, California 92037, USA,<br> 3. Keck School of Medicine, University of Southern California, Los Angeles, California 90033, USA,<br>
4. Viterbi School of Engineering, University of Southern California, Los Angeles, California 90089, USA</span></td>
		</tr>
		<tr>
			<td>Journal</td>
			<td>Cold Spring Harbor Molecular Case Studies</td>
		</tr>
		<tr>
			<td>Publication URL</td>
			<td>http://molecularcasestudies.cshlp.org/</td>
		</tr>
		<tr>
			<td>Data Availability Date</td>
			<td></td>
		</tr>
		<tr>
			<td>Brief Description of Data Available</td>
			<td>low-pass copy number alternation (CNA) data from single cells and cell-free DNA (cfDNA), targeted sequencing of cfDNA, whole-exome sequencing and Sanger sequencing of the primary breast tissue and metastatic bone and liver biopsies, single-cell Sanger sequencing</td>
		</tr>
		<tr>
			<td>Sponsors/Acknowledgements</td>
			<td></td>
		</tr>
		<tr>
			<td>Study number/codes</td>
			<td></td>
		</tr>
		<tr>
			<td>Condition</td>
			<td>Metastatic Breast Cancer</td>
		</tr>
		<tr>
			<td>Study Type</td>
			<td></td>
		</tr>
		<tr>
			<td>Websites</td>
			<td></td>
		</tr>
	</tbody>
</table>
  <h2>Open Access Links</h2>
  <p>
      Open access data (with authz set to "*") can download without authentication.
  </p>

  <button type="button" data-gen3-did="0a2ea71e-6f43-4588-8703-361cfe67add4" class="g3-button_action">CNA Data</button>
  <br>
  <p>fastq files of low pass whole genome sequencing data for copy number alteration (CNA) analysis of FFPE tissue, cfDNA and single cells</p>
  <br>
  <button type="button" data-gen3-did="0a2ea71e-6f43-4588-8703-361cfe67add4" class="g3-button_action">cfDNA Targeted Sequencing</button>
  <br>
  <p>13 bam files of cfDNA samples sequenced using the ThermoFisher Oncomine Breast cfDNA Assay v2</p>
  <br>
  <button type="button" data-gen3-did="0a2ea71e-6f43-4588-8703-361cfe67add4" class="g3-button_action">Sanger Sequencing</button>
  <br>
  <p>ab1 trace files of FFPE tissue from primary breast, bone metastasis and liver metastasis for ESR1; ab1 trace files of single CTCs  for ESR1 and PIK3CA</p>
  <br>
  <button type="button" data-gen3-did="0a2ea71e-6f43-4588-8703-361cfe67add4" class="g3-button_action">WES Data</button>
  <br>
  <p>fastq files of paired-end whole exome sequencing runs. 2 separate runs per sample. Sample types are primary breast tissue, bone metastasis tissue and liver metastasis tissue</p>
  <script src="./download-helper.js"></script>
</body>
</html>
